packages/shared-metrics/src/libuv.js

```diff
@@ -9,9 +9,13 @@ const path = require('path');
 
 const lag = require('event-loop-lag')(1000);
 
+/** @type {*} */
 let eventLoopStats;
 
 exports.payloadPrefix = 'libuv';
+
+// This line can probly be removed. On the other hand, this is a 6 year old code, so we should review it anyway.
```

Comment:
@willianpc I don't really understand what this comment is about.

Reply:
Sorry, that was not clear at all, and I may either remove the comment or make it as a TODO and add more info.

So, TS complains that this object is read only (check line 40), thus, it cannot be assigned a value.
What I had in mind was that maybe we could review the code together at some point and see if the `defineProperty` is still needed, and if so, if we can remove the "forbidden" assignment of `{}` to the object.
For the moment, I am just ignoring the warning and trying to focus on the typing only. Even though I changed some code, I try my best to avoid it at this stage.

Is it more clear?

Reply:
Absolutely 👍   Thanks for the explanation.